ogr/services/github.py

```diff
@@ -102,6 +106,13 @@ def __init__(
             )
         self.read_only = read_only
 
+    def __str__(self) -> str:
+        return (
+            f"Github-Project"
+            f"namespace: '{self.namespace}'"
+            f"repo: '{self.repo}'"
```

Comment:
this won't look pretty, please wrap the parameters in brackets: `GithubProject(namespace: 'asd', ...)` and so on

Reply:
Yes, I will make the changes. Thanks for feedback !